src/interface/sparse_pack.hpp

```diff
@@ -247,6 +247,18 @@ class SparsePack : public SparsePackBase {
     return bounds_h_(1, b, vidx);
   }
 
+  template <class... Args>
+  KOKKOS_INLINE_FUNCTION bool Exists(const int b, Args... args) const {
```

Comment:
Does this need to be a variadic template if we're (presumably) only checking for the existence of one var at a time? Will
```c++
template <typename T>
KOKKOS_INLINE_FUNCTION bool Exists(const int b, T var) const {
  // etc.
}
```
not work?

Reply:
You're totally right, that would work. I somehow thought the topological element was an argument to `GetUpperBound` but it's not. I can make it a variatic pack still so you can check if the pack contains *anything* (i.e., if you pass in zero arguments) or I can just make it a template on one var. What do you think?

Reply:
Following @lroberts36 suggestion, this now can return the existence status of multiple vars at once via the pack. Either with `Contains<v1, v3, v5>(b)` or with `Contains(b, v1(), v3(), v5())`